service/dependency.go

```diff
@@ -36,7 +36,7 @@ type Dependency struct {
 // error logs.
 func (d *Dependency) Logs() (rstd, rerr io.ReadCloser, err error) {
 	var reader io.ReadCloser
-	reader, err = defaultContainer.ServiceLogs(d.namespace())
+	reader, err = d.service.docker.ServiceLogs(d.namespace())
```

Comment:
Why not injecting docker into `Dependency` like that we don't need to use `.service.docker` and we remove one dependency

Reply:
we're reading some other _Service_ fields in other places too. I'm creating another issue to getting ridding of these cycling references. https://github.com/mesg-foundation/core/issues/475

Reply:
👍 let's not block for that, good to track it in an issue